interfaces.go

```diff
@@ -15,15 +15,37 @@ type Connection interface {
 
 // KeySpace is used to obtain tables from.
 type KeySpace interface {
-	MapTable(tableName, id string, row interface{}) MapTable
-	MultimapTable(tableName, fieldToIndexBy, uniqueKey string, row interface{}) MultimapTable
-	MultimapMultiKeyTable(tableName string, fieldToIndexBy, uniqueKey []string, row interface{}) MultimapMkTable
-	TimeSeriesTable(tableName, timeField, uniqueKey string, bucketSize time.Duration, row interface{}) TimeSeriesTable
-	MultiTimeSeriesTable(tableName, fieldToIndexByField, timeField, uniqueKey string, bucketSize time.Duration, row interface{}) MultiTimeSeriesTable
-	MultiKeyTimeSeriesTable(tableName string, fieldToIndexByField []string, timeField string, uniqueKey []string, bucketSize time.Duration, row interface{}) MultiKeyTimeSeriesTable
-	FlakeSeriesTable(tableName, idField string, bucketSize time.Duration, row interface{}) FlakeSeriesTable
-	MultiFlakeSeriesTable(tableName, indexField, idField string, bucketSize time.Duration, row interface{}) MultiFlakeSeriesTable
-	Table(tableName string, row interface{}, keys Keys) Table
+	// MapTable is a simple key-value store.
+	MapTable(prefixForTableName, partitionKey string, rowDefinition interface{}) MapTable
+	// MultimapTable lets you list rows based on a field equality, eg. 'list all sales where seller id = v'.
+	// It uses the partitionKey for partitioning the data. The ordering within a partition is determined using the clusteringKey.
+	MultimapTable(prefixForTableName, partitionKey, clusteringKey string, rowDefinition interface{}) MultimapTable
+	// // MultimapMkTable lets you list rows based on several fields equality, eg. 'list all sales where seller id = v and name = 'john'.
```

Comment:
We can improve the descriptions for `MultimapTable` and `MultimapMultiKeyTable`. This is because `MultimapTable` does let you list rows based on several fields equality already, but it's restricted to one partition key, whereas `MultimapMultiKeyTable` does not.

We should also update this comment so `MultimapMkTable` matches `MultimapMultiKeyTable`